copt/constraint.py

```diff
@@ -47,23 +47,20 @@ def lmo_pairwise(self, u, x):
         )
 
         u_active = -u * np.sign(x)
-        ma_u_active = ma.array(u_active, mask=(u_active == 0))
+        ma_u_active = ma.array(u_active, mask=(x== 0) + (u_active==0))
```

Comment:
I'm not entirely convinced by this. What if the domain does not include zero? I think the issue has more to do with how to include x0 as a vertex so we start with atom in the support than with zero